### PR TITLE
Search

### PR DESCRIPTION
**Pull request by kosovych**

### CODE REVIEW

**Comment by dymytrii-tretiakov** [comment]
Do you try use react and  redux devtools in project?

File: jsconfig.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+    "paths": {
```
Comment (dymytrii-tretiakov) [inline jsconfig.json:4]: 👍

File: src/components/shared/Empty/index.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    textAlign: 'center',
+  },
+});
+
+const Empty = () => {
+  return (
+    <View style={styles.wrapper}>
+      <Image
+        style={styles.img}
+        source={require('../../../../public/images/empty.png')}
```
Comment (dymytrii-tretiakov) [inline src/components/shared/Empty/index.js:29]: may you add short path to the image folder?

File: src/schemas/searchMoviesSchema.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import {schema} from 'normalizr';
```
Comment (dymytrii-tretiakov) [inline src/schemas/searchMoviesSchema.js:1]: do you know why need a `normalizr` ?

File: src/components/shared/SearchInput/index.js [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+    height: 20,
+    marginRight: 6,
+  },
+  input: {
+    flexGrow: 1,
+    padding: 4,
+    fontSize: 16,
+  },
+  errorInput: {
+    borderColor: '#dd1f00',
```
Comment (dymytrii-tretiakov) [inline src/components/shared/SearchInput/index.js:42]: maybe create theme file in constants and move all hex to them?

**Review by dymytrii-tretiakov — CHANGES_REQUESTED** [review]

File: src/schemas/searchMoviesSchema.js (hunk 3 above)
Comment (kosovych) [reply src/schemas/searchMoviesSchema.js:1]: yes

**Pull request merged**